META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,21 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Functional Java
+Bundle-SymbolicName: org.functionaljava
+Bundle-Version: 4.3.0.qualifier
```

Comment:
Hmm it's a good point
Also I have since learned that gradle is generating the manifest
currently.  I should redo this one.
On Jan 4, 2015 5:02 PM, "Mark Perry" <notifications@github.com> wrote:

> In META-INF/MANIFEST.MF
> <https://github.com/functionaljava/functionaljava/pull/72#discussion-diff-22444882>
> :
>
> > @@ -0,0 +1,21 @@
> > +Manifest-Version: 1.0
> > +Bundle-ManifestVersion: 2
> > +Bundle-Name: Functional Java
> > +Bundle-SymbolicName: org.functionaljava
> > +Bundle-Version: 4.3.0.qualifier
>
> @dobesv <https://github.com/dobesv> I don't know about Eclipse's use of
> manifests, but should the bundle version above be 4.2 for the last release?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/functionaljava/functionaljava/pull/72/files#r22444882>
> .
>